pkg/server/test/list_objects.go

```diff
@@ -27,7 +27,9 @@ const (
 	defaultListObjectsDeadline   = 5 * time.Second
 	defaultListObjectsMaxResults = 5
 
-	checkConcurrencyLimit = 100
+	checkConcurrencyLimit   = 100
+	defaultResolveNodeLimit = 25
```

Comment:
these const. were in the test/check.go and used by this file and other test file.

Reply:
Let's just add these in `pkg/server/test/server.go` instead. They can/may be used across test files.

Reply:
Done